snuba/query/parser/__init__.py

```diff
@@ -224,3 +240,186 @@ def apply_aliases(exp: Expression) -> Expression:
             return replace(exp, alias=exp.column_name)
 
     query.transform_expressions(apply_aliases)
+
+
+def _expand_aliases(query: Query) -> None:
```

Comment:
See https://github.com/getsentry/snuba/pull/1024/files#diff-dd1e177b6b14ec2cdec36b243782b920R200 to see how this works in practice.